backend/src/services/linear/api/resolvers.py

```
@@ -15230,3 +15453,487 @@ def resolve_teamMembershipUpdate(obj, info, **kwargs):
+                query = query.join(Team, WorkflowState.teamId == Team.id)
+            query = apply_team_filter(query, team_filter)
+
+    if "issues" in filter_dict:
+        issues_filter = filter_dict["issues"]
+        if issues_filter and isinstance(issues_filter, dict):
+            # For collection filters, we need to check if ANY issue matches
+            # This requires EXISTS subquery
+            from sqlalchemy import exists
+
+            # Create correlated subquery
+            issue_exists = exists().where(Issue.stateId == WorkflowState.id)
```

Ruff (0.14.1)

Local variable `issue_exists` is assigned to but never used

Remove assignment to unused variable `issue_exists`

(F841)